include/bluetooth/direction.h

```diff
@@ -217,4 +230,21 @@ int bt_df_conn_cte_rx_enable(struct bt_conn *conn, const struct bt_df_conn_cte_r
  */
 int bt_df_conn_cte_rx_disable(struct bt_conn *conn);
 
+/**
+ * @brief Set Constant Tone Extension transmission parameters for a connection.
+ *
+ * The function is available if Kconfig option CONFIG_BT_DF_CONNECTION_CTE_TX is enabled.
+ *
+ * @note If bt_df_conn_cte_tx_param.cte_types bits BT_DF_CTE_TYPE_AOD_1US and BT_DF_CTE_TYPE_AOD_2US
```

Comment:
Does `bt_df_conn_cte_tx_param.cte_types` do a proper reference to the field, or is it missing some `@ref`? I feel like remembering that to reference a field in a struct, you need to do something like `@ref struct_name::field_name`

Reply:
This applies multiple places here.

Alternative, you can also reference the `params` parameter instead of the struct, using something like `@p params.cte_types` I think

Reply:
![Screenshot from 2021-12-28 20-09-45](https://user-images.githubusercontent.com/53781024/147599215-26ba6bdf-d18d-44d5-a084-acedd74d6ba9.png)
Doxygen correcly creates a reference and link documentation to a member of struct.
I'll add and information in the note that it refers to `param`